runtime/makelib/targets.mk.ftl

```diff
@@ -513,7 +519,7 @@ MINGW_CXXFLAGS+=-std=c++0x -D_CRT_SUPPRESS_RESTRICT -DVS12AndHigher
 	MINGW_CXXFLAGS+=-D_M_X64
 </#if>
 endif
-# special handling BytecodeInterpreter.cpp and DebugBytecodeInterpreter.cpp
+# JIT helper methods require us to disable buffer security checks on Windows
```

Comment:
This comment change should be reverted as well.

Reply:
@pshipton , misread 'changes' view.  Should be good now, also rebased.